templates/frontend/objects/article_details.tpl

```diff
@@ -215,10 +215,10 @@
 				<div class="item cover_image">
 					<div class="sub_item">
 						{if $article->getLocalizedCoverImage()}
-							<img src="{$article->getLocalizedCoverImageUrl()|escape}"{if $article->getLocalizedCoverImageAltText()} alt="{$article->getLocalizedCoverImageAltText()|escape}"{/if}>
+							<img src="{$article->getLocalizedCoverImageUrl()|escape}" alt="{$article->getLocalizedCoverImageAltText()|default:'null'}">
```

Comment:
Ah, so I didn't notice before, but here in these OJS changes there is the same issue with escape.

Reply:
Ok!